### ISSUE TITLE
calibration broke between version 201 and 260

### ISSUE THREAD

**Issue by trishorts** [smith-chem-wisc/MetaMorpheus#1002]
calibration versus time especially bad

**Comment by trishorts** [comment]
I was able to successfully search a file calibrated with version 201 using version 260. That is an option for some people.

**Comment by zrolfs** [comment]
Does 201 generate file specific tolerances?

**Comment by rmillikin** [comment]
I don't think so, but you can just do it manually by using 3* the std dev of the ppm mass error or whatever you want to use

**Comment by rmillikin** [comment]
I'll add also, if you encounter data where an older version of MM does a better job calibrating, please send it to me. I'd like to make sure the latest version of MM is the best in all respects.

**Comment by zrolfs** [comment]
@trishorts What were the issues that you were encountering? Is it that the calibration is slower now than before?
@rmillikin Are the suggested tolerances still the optimal search tolerances? They were not when analyzing dmorgen's data.

**Comment by rmillikin** [comment]
Have not investigated that yet. I'd like to use the Jurkat dataset that Stefan used for evaluating calibration because he observed that a couple of those files had much larger PPM errors after calibration compared to the others, so I want to make sure it's now consistent across several files. This issue is simply for evaluating whether or not calibration is still working, broadly speaking

**Comment by trishorts** [comment]
i have a jurkat file that doesn't look like it calibrates on the time axis. will try again. 
![image](https://user-images.githubusercontent.com/16841846/37110849-b806f8ac-2203-11e8-8a53-4c25c9f5cccf.png)

**Comment by trishorts** [comment]
confirmed with jurakat file and fasta in newest release
